packages/google_sign_in/lib/google_sign_in.dart

```diff
@@ -128,7 +128,7 @@ class GoogleSignIn {
       );
     }
     await _initialization;
-    Map<String, dynamic> response = await _channel.invokeMethod(method);
+    Map<String, dynamic> response = await _channel.invokeMethod(method,null);
```

Comment:
This is to get Mockito not ignore this call because of the missing optional positional argument. I solved this with a change in the test in #93.